modules/apps/password-policies-admin/password-policies-admin-web/src/main/java/com/liferay/password/policies/admin/web/internal/display/context/PasswordPolicyDisplayContext.java

```diff
@@ -160,9 +175,11 @@ public boolean hasAssignMembersPermission() {
 
 		return PasswordPolicyPermissionUtil.contains(
 			themeDisplay.getPermissionChecker(),
-			passwordPolicy.getPasswordPolicyId(), ActionKeys.ASSIGN_MEMBERS);
+			passwordPolicy.getPasswordPolicyId(), actionId);
 	}
 
+	private PasswordPolicy _passwordPolicy;
+	private long _passwordPolicyId;
```

Comment:
So according to the comment above, this line would read `private Long _passwordPolicyId;`